templates/addons/aws/modules/vpc_flow_log/main.tf

```diff
@@ -0,0 +1,296 @@
+data "aws_caller_identity" "current" {}
+data "aws_partition" "current" {}
+data "aws_region" "current" {}
+
+locals {
```

Comment:
I think we can now move locals and data into separate files to not fatten this file